beta/nncf/tensorflow/graph/utils.py

```diff
@@ -109,6 +109,10 @@ def get_original_name_and_instance_index(node_name):
     return original_name, instance_index
 
 
+def get_original_name(node_name):
```

Comment:
I have added attribute 'original_name' to nncfnode. 
Now it's looks useless. Could you remove https://github.com/openvinotoolkit/nncf/blob/develop/beta/nncf/tensorflow/graph/converter.py#L250?

Reply:
@AlexanderDokuchaev Sequential models don't have this attribute

Reply:
I sagest to remove original_name from node attribute  `graph.add_node(node_name, **attributes)`. You replaced all cases of using of `node.data['original_name']` to get_original_name. As result this attribute can be remove.

Reply:
@AlexanderDokuchaev We decided to open an issue on the interconnection of shared nodes in a model and in a graph. I think it better to change this in the upcoming pr.